oi/CodeGen.cpp

```diff
@@ -398,6 +412,140 @@ void addGetSizeFuncDefs(const TypeGraph& typeGraph,
     }
   }
 }
+
+void addStandardTypeHandlers(std::string& code) {
+  code += R"(
+    template <typename DB, typename T>
+    StaticTypes::Unit<DB>
+    getSizeType(const T &t, typename TypeHandler<DB, T>::type returnArg) {
+      JLOG("obj @");
+      JLOGPTR(&t);
+      return TypeHandler<DB, T>::getSizeType(t, returnArg);
+    }
+  )";
+
+  code += R"(
+    template<typename DB, typename T0, long unsigned int N>
```

Comment:
Why "T0" instead of "T"?

Reply:
The idea was to name all the template parameters in these `TypeHandler` classes consistently so they can be specified in a more abstract way in the future in the `.toml` files, and potentially carry that into this piece of code. Looking at this now `N` probably needs to be either `T1` or `N0`.